README.md

```diff
@@ -13,34 +13,33 @@ Currently supports
  - Coub
  - Wistia
 
-# grunt
-
-```shell
-$ grunt
-$ grunt build
-$ grunt test
-$ grunt dist
-
-#build and test specific plugins
-$ grunt --youtube --vimeo
-$ grunt build --youtube --vimeo
-$ grunt test --youtube --vimeo
-$ grunt dist --youtube --vimeo
+# npm
+```
+npm install js-video-url-parser
 ```
 
 # bower
 ```shell
 bower install js-video-url-parser
 ```
-# npm
-```
-npm install js-video-url-parser
-```
 
 # Usage
-## Node
+## ES2015+ / Webpack
 ```
-urlParser = require('js-video-url-parser');
+// All plugins
+import urlParser from "js-video-url-parser";
+
+// Choose individual plugins
+import urlParser from "js-video-url-parser/lib/base";
+import "js-video-url-parser/lib/provider/canalplus';
+import "js-video-url-parser/lib/provider/coub';
+import "js-video-url-parser/lib/provider/dailymotion';
+import "js-video-url-parser/lib/provider/twitch';
+import "js-video-url-parser/lib/provider/vimeo';
+import "js-video-url-parser/lib/provider/wistia';
+import "js-video-url-parser/lib/provider/youku';
```

Comment:
Does this put the providers in the global namespace? I've had an issue before with a namespace clash (#25).

Reply:
It does not. This is the ES2015 import syntax, similar to node's require. Each `.js` file is a separate isolated module.